lib/engine/game/g_18_cz/game.rb

```diff
@@ -325,6 +325,21 @@ def timeline
           @timeline.append("Next set of Operating Rounds will have #{OR_SETS[@turn - 1]} ORs")
         end
 
+        def train_type_by_name(name)
```

Comment:
There is a function to map train to corp type. Im just mobile so I cannot link direkt, but toby will not approve regexes

Reply:
Thanks, I found the TRAINS_FOR_CORPORATIONS map